long-running/jaxrs-datagrid-hibernate/src/test/java/io/quarkus/ts/resources/ContactResourceIT.java

```diff
@@ -0,0 +1,103 @@
+package io.quarkus.ts.resources;
+
+import static io.restassured.RestAssured.given;
+import static org.awaitility.Awaitility.await;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.TestInstance.Lifecycle;
+
+import javax.ws.rs.core.MediaType;
+
+import org.apache.http.HttpStatus;
+import org.jboss.logging.Logger;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.ts.memoryLeaks.MemoryManager;
+import io.quarkus.ts.memoryLeaks.MemoryReport;
+import io.quarkus.ts.model.Contact;
+import io.restassured.response.ValidatableResponse;
+
+@QuarkusScenario
```

Comment:
Executed application should have set -Xmx and -Xms, both set to the same value.

There are several reasons:
 - no cpu / time spend on heap increase
 - constrained and stable memory setting